### ISSUE TITLE
Clicking "Edit code" opens an editor with no code

### ISSUE THREAD

**Issue by alphalambda** [google/codeworld#395]
I was expecting "Edit code" to open an editor pre-filled with the code I was creating in the blocks interface, but instead I get an empty editor. I noticed I can cut and paste the code from the blocks window into the regular CodeWorld editor, but it would be nicer if the editor were pre-filled with my code.

**Comment by cdsmith** [comment]
Yeah, this is broken.  It works if you click Run first.  What the button should do is first transfer your code to the server and get a hash if needed, and then open the window with the appropriate hash, so the new window will load the code from the server.

If you click Run first, then it works as it should, because there's already a hash associated with your code.
